conf/services/open-banking.yml

```diff
@@ -61,4 +61,5 @@ milestones:
 serviceLastTestedDate: 2021-03-12
 statementVisibility: public
 statementCreatedDate: 2021-03-24
-statementLastUpdatedDate: 2021-03-21
\ No newline at end of file
+statementLastUpdatedDate: 2021-03-24
```

Comment:
This `statementLastUpdatedDate` update also looks like it's on the wrong file, shouldn't it have been changed in direct-debit.yml?